benchmark/pom.xml

```diff
@@ -151,4 +152,23 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>default</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <properties>
+                <minimize>false</minimize>
+            </properties>
+        </profile>
+        <profile>
+            <id>blazemeter</id>
```

Comment:
We do not utilize Blaze-meter tooling in our Keycloak Benchmark, @lvassa88 can you please elaborate how would this be useful for the folks who do not use the Blaze-meter in their frameworks ?

Reply:
I try to use the keycloak benchmark jar on blameter to run test in different situation/timezone, but the jar that you generate is bigger for BlazeMeter tool, so I use the minimizeJar to solve the issue and upload the jar generated with this option and I think other guys can have the same problem.